go/test/endtoend/vtorc/utils/utils.go

```diff
@@ -582,11 +582,29 @@ func RunSQL(t *testing.T, sql string, tablet *cluster.Vttablet, db string) (*sql
 	defer conn.Close()
 
 	// RunSQL
-	return execute(t, conn, sql)
+	return RunSQLWithSuperReadOnly(t, sql, tablet, db, false)
```

Comment:
I think this is another place that leaves us, in the end, w/o much testing of this work.

Reply:
RunSQLWithSuperReadOnly is passed false in the last argument for super_read_only.